### ISSUE TITLE
AttributeError: 'TensorVariable' object has no attribute 'get_shape'

### ISSUE THREAD

**Issue by arogozhnikov** [bethgelab/foolbox#45]
Hi, after running an example in readme I get:
```

AttributeErrorTraceback (most recent call last)
<ipython-input-19-74364fe58d77> in <module>()
      8 kmodel = ResNet50(weights='imagenet')
      9 preprocessing = (numpy.array([104, 116, 123]), 1)
---> 10 fmodel = foolbox.models.KerasModel(kmodel, bounds=(0, 255), preprocessing=preprocessing)
     11 
     12 # get source image and label

/usr/local/lib/python2.7/dist-packages/foolbox/models/keras.pyc in __init__(self, model, bounds, channel_axis, preprocessing, predicts)
     59             predictions_are_logits = True
     60 
---> 61         shape = predictions.get_shape().as_list()
     62         _, num_classes = shape
     63         assert num_classes is not None

AttributeError: 'TensorVariable' object has no attribute 'get_shape'
```

My setup:
```
Keras==2.0.6
Theano=0.9.0
```
current foolbox from pip

**Comment by wielandbrendel** [comment]
Hey @arogozhnikov, so you are using Keras with the Theano backend, right? We are currently only testing with Tensorflow backend. I'll take a look at a fix.

**Comment by wielandbrendel** [comment]
The PR https://github.com/bethgelab/foolbox/pull/46 should fix this issue, @jonasrauber is going to merge it soon.

Before closing this issue though we should extend pytests to include the different Keras backends (maybe even CNTK?). @jonasrauber : Can we run pytests on specific files with different environmental variables (we'd have to set `os.environ['KERAS_BACKEND'] = backend`)?

**Comment by jonasrauber** [comment]
Closing this because this issue should be solved in Foolbox 0.9.2. Created a new issue for the keras backend testing.
